astropy/modeling/mappings.py

```diff
@@ -122,6 +125,8 @@ def inverse(self):
         inv = self.__class__(mapping)
         inv._inputs = self._outputs
         inv._outputs = self._inputs
+        inv._n_inputs = len(inv._inputs)
+        inv._n_outputs = len(inv._outputs)
         return inv
```

Comment:
So this model doesn't support .N_inputs and .N_outputs? If not, should it?

Reply:
See comment above.